usaspending_api/api_contracts/contracts/v2/search/spending_by_category/country.md

```diff
@@ -0,0 +1,166 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Country [/api/v2/search/spending_by_category/country/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## POST
+
+This endpoint returns a list of the top results of Countries sorted by the total amounts in descending order.
+
++ Request (application/json)
+    + Attributes (object)
+        + `filters` (required, FilterObject)
+            The filters to find with said category
+        + `limit`: 5 (optional, number)
+            The number of results to include per page
+        + `page`: 1 (optional, number)
+            The page of results to return based on the limit
+        + `subawards` (optional, boolean)
+            Determines whether Prime Awards or Sub Awards are searched
+    + Body
```

Comment:
I realize I'm late to the party.

I'm surprised the Request contains a Body yet the Response does not. I was under the impression the Frontend team desired a Response.Body JSON object for their mock server.

Is that correct @ebdabbs & @jonhill13 ?

Reply:
I can add a response body to the contract, if that's what frontend wants.

Reply:
I asked @jonhill13 about it and he was fine with it as-is, but if anyone feels strongly about it I'm open to adding a response

Reply:
I do not think I am qualified to respond about our mock server, since I have yet to use it. But I can say that I have approved other new contracts for ES that do not have Response.body

Reply:
@tony-sappe haven't used the mock server in 6+ months, but I am in favor of consistency, and that appears to be -- to my understanding -- what we're doing here. 

https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/awards/funding_rollup.md

⬆️ that's what I've used it for and I believe we are expecting a body.

If I'm mistaken that omitting `response.body` would be inconsistent, I'm fine with omitting it.

Reply:
I think it's going to take less time for us to just copy/paste a body in to each of these than try to find the authoritative rule...

Reply:
The best documentation I can find from our meetings July 9th and 10th, 2019 is this API contract example:
https://github.com/fedspendingtransparency/usaspending-api/blob/proposed-api-contract-template/usaspending_api/api_contracts/api_blueprint_contract_template.md

Response.Body is present, Request.Body is not

Reply:
I've added Response.body to the contract.